chia/wallet/wallet_puzzle_store.py

```diff
@@ -39,11 +39,12 @@ async def create(cls, db_wrapper: DBWrapper2, cache_size: uint32 = uint32(600000
                     "CREATE TABLE IF NOT EXISTS derivation_paths("
                     "derivation_index int,"
                     " pubkey text,"
-                    " puzzle_hash text PRIMARY KEY,"
```

Comment:
Did we ever release the table version with `puzzle_hash text PRIMARY KEY`?

If so, we'll have to do a DB migration for clients that already have the `derivation_paths` table.

Reply:
That's what's on `main` for quite awhile and multiple releases, yes.  It was only discovered as a bug because data layer had another bug where it reused other wallet's puzzle hashes.  This has been discussed and deemed ok as is.  Nobody _needs_ the change.  When they rebuild their db they will get the change.  If we actually need this in the future we can do an official migration.